src/Microsoft.Performance.SDK.Runtime/Extensibility/DataExtensions/Tables/TableExtensionReference.cs

```diff
@@ -203,14 +183,6 @@ public bool IsInternalTable
                     out var tableBuildAction,
                     out var tableIsDataAvailableFunc))
             {
-                if (tableDescriptor.IsInternalTable)
-                {
-                    if (!allowInternalTables)
-                    {
-                        return false;
-                    }
-                }
-
```

Comment:
I upgraded WPA and XPerf with your changes, found that processor specified tables were being added as external. 

Updating to check if the build table action is null should work

Reply:
This should be fixed now. Please let me know if there are still issues.